apps/performance/ngfor-optimize/src/app/components/person-list.component.ts

```diff
@@ -34,4 +34,8 @@ export class PersonListComponent {
   @Input() persons: Person[] = [];
   @Output() delete = new EventEmitter<string>();
   @Output() update = new EventEmitter<string>();
+
+  trackByFn(index: number, person: Person) {
+    return person.email;
+  }
```

Comment:
👍

You can now use the new control flow if you want 🔥

Reply:
Angular17, Ngrx17 very tempting, but i have to survive